src/plugins/maps_legacy/README.md

```diff
@@ -0,0 +1,3 @@
+# Maps legacy
+
+Internal objects used by the Coordinate, Region, and Vega visualizations. This contains the default Leaflet-based map.
```

Comment:
Can you add something about exporting EMS client used by Coordinate, Region, and Vega visualizations